torch/_inductor/config.py

```diff
@@ -45,7 +45,7 @@ def is_fbcode():
 )
 
 c_shim_version = os.environ.get(
-    "TORCHINDUCTOR_C_SHIM_VERSION", "1" if is_fbcode() else "2"
+    "TORCHINDUCTOR_C_SHIM_VERSION", "1" if (is_fbcode() and torch.version.hip) else "2"
```

Comment:
We should get rid of `torch.version.hip`, but let's chat offline to figure out how to do it.